gnupg/_util.py

```diff
@@ -126,7 +126,7 @@
 log = _logger.create_logger(0)
 
 #: Compiled regex for determining a GnuPG binary's version:
-_VERSION_STRING_REGEX = re.compile('(\d)*(\.)*(\d)*(\.)*(\d+)*')
```

Comment:
what's the bug here exactly?
If I'm not mistaken, gnupg version in ubuntu 17.04 is 2.1.15, and the original regex seems to capture that properly, isn't it?